### ISSUE TITLE
Update tap output when grpc-status is missing

### ISSUE THREAD

**Issue by seanmonstar** [runconduit/conduit#198]
The [tap output](https://github.com/runconduit/conduit/blob/master/cli/cmd/tap.go#L170) includes the `grpc-status` for all `ResponseEnd` events. HTTP/1 and any HTTP/2 that isn't gRPC traffic will not include a grpc-status. The output for tap should probably not show:

```
end id=1:1 src=8.8.8.8:3001 dst=8.8.8.8:3000 grpc-status=OK duration=10µs response-length=0B
```

The missing code means that `0` will be returned, which gets output as `OK`, but it's kind of a lie, since there actually was no `grpc-status` at all.

**Comment by hawkw** [comment]
@siggy I went ahead and assigned myself to this as well, since we determined it will also require some proxy changes. Would be happy to break this into separate proxy/cli issues if you think that makes more sense?

**Comment by siggy** [comment]
@hawkw proto branch is up: https://github.com/runconduit/conduit/tree/siggy/tap-grpc-option

go ahead and break the proxy work into a separate issue, assuming we can do this work in parallel.

**Comment by hawkw** [comment]
@siggy excellent, shall. As far as I know, now that you've made the common proto changes, we should be able to.

**Comment by hawkw** [comment]
Okay, created #202.

**Comment by hawkw** [comment]
I believe this was closed by #204?
